### PR TITLE
[Dungeon] Zul'farrak

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Dungeon.
Added flags for each boss, so they will appear with skull instead regular level elite.
Missing AI included now as well.

Spawns and AI for extra 3 rares completed.
They will appear like any other Rare that we have in Dungeons now.
Only path need to be updated, but will add that once im finished with other dungeons.

All checks have been done trough Classic Realms and Youtube videos.

How to test:
Apply SQL
Restart Server and delete your Cache.

Make sure to add very very basic gear towards each Dungeon.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1588338165355317100.sql [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+(205800, 10081, 209, 0, 0, 1, 1, 0, 1, 1863.89, 1151.66, 12.9287, 0.540263, 86400, 0, 0, 5757, 0, 0, 0, 0, 0, '', 0);
+
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES 
+(205801, 10082, 209, 0, 0, 1, 1, 0, 1, 1674.24, 904.853, 8.92432, 0.304632, 86400, 0, 0, 5544, 0, 0, 0, 0, 0, '', 0);
+
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES 
+(205802, 10080, 209, 0, 0, 1, 1, 0, 1, 1546.16, 1015.47, 8.87699, 0.202525, 86400, 0, 0, 5544, 0, 0, 0, 0, 0, '', 0);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588338165355317100.sql:82]:
```suggestion
DELETE FROM `creature` WHERE `guid` IN (205800, 205801, 205802);
INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES 
(205800, 10081, 209, 0, 0, 1, 1, 0, 1, 1863.89, 1151.66, 12.9287, 0.540263, 86400, 0, 0, 5757, 0, 0, 0, 0, 0, '', 0),
(205801, 10082, 209, 0, 0, 1, 1, 0, 1, 1674.24, 904.853, 8.92432, 0.304632, 86400, 0, 0, 5544, 0, 0, 0, 0, 0, '', 0),
(205802, 10080, 209, 0, 0, 1, 1, 0, 1, 1546.16, 1015.47, 8.87699, 0.202525, 86400, 0, 0, 5544, 0, 0, 0, 0, 0, '', 0);
```

File: data/sql/updates/pending_db_world/rev_1588338165355317100.sql [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 172, `maxdmg` = 228, `DamageModifier` = 1.01 WHERE `entry` = 8127; 
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 172, `maxdmg` = 228, `DamageModifier` = 1.01 WHERE `entry` = 7271; 
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 182, `maxdmg` = 242, `DamageModifier` = 1.01 WHERE `entry` = 7604; 
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 131, `maxdmg` = 183, `DamageModifier` = 1.01 WHERE `entry` = 7795; 
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 269, `maxdmg` = 346, `DamageModifier` = 1.01 WHERE `entry` = 7796; 
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 259, `maxdmg` = 334, `DamageModifier` = 1.01 WHERE `entry` = 7275; 
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 251, `maxdmg` = 325, `DamageModifier` = 1.01 WHERE `entry` = 7273;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 237, `maxdmg` = 314, `DamageModifier` = 1.01 WHERE `entry` = 7797; 
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 237, `maxdmg` = 314, `DamageModifier` = 1.01 WHERE `entry` = 7267;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588338165355317100.sql:48]:
```suggestion
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 233, `maxdmg` = 309, `DamageModifier` = 1.01 WHERE `entry` = 7272; 
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 172, `maxdmg` = 228, `DamageModifier` = 1.01 WHERE `entry` IN (8127, 7271); 
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 182, `maxdmg` = 242, `DamageModifier` = 1.01 WHERE `entry` = 7604; 
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 131, `maxdmg` = 183, `DamageModifier` = 1.01 WHERE `entry` = 7795; 
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 269, `maxdmg` = 346, `DamageModifier` = 1.01 WHERE `entry` = 7796; 
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 259, `maxdmg` = 334, `DamageModifier` = 1.01 WHERE `entry` = 7275; 
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 251, `maxdmg` = 325, `DamageModifier` = 1.01 WHERE `entry` = 7273;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 237, `maxdmg` = 314, `DamageModifier` = 1.01 WHERE `entry` IN (7797, 7267); 
```

File: data/sql/updates/pending_db_world/rev_1588338165355317100.sql [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+UPDATE `creature_template` SET `mindmg` = 141, `maxdmg` = 187, `DamageModifier` = 1.03 WHERE `entry` = 7605;
+UPDATE `creature_template` SET `mindmg` = 167, `maxdmg` = 213, `DamageModifier` = 1.03 WHERE `entry` = 7608;
+UPDATE `creature_template` SET `mindmg` = 122, `maxdmg` = 170, `DamageModifier` = 1.03 WHERE `entry` = 7606;
+UPDATE `creature_template` SET `mindmg` = 152, `maxdmg` = 198, `DamageModifier` = 1.03 WHERE `entry` = 7789;
+UPDATE `creature_template` SET `mindmg` = 73, `maxdmg` = 96, `DamageModifier` = 1.03 WHERE `entry` = 7788;
+UPDATE `creature_template` SET `mindmg` = 88, `maxdmg` = 99, `DamageModifier` = 1.03 WHERE `entry` = 8877;
+UPDATE `creature_template` SET `mindmg` = 77, `maxdmg` = 98, `DamageModifier` = 1.03 WHERE `entry` = 7787;
+UPDATE `creature_template` SET `mindmg` = 58, `maxdmg` = 87, `DamageModifier` = 1.03 WHERE `entry` = 8876;
+UPDATE `creature_template` SET `mindmg` = 182, `maxdmg` = 242, `DamageModifier` = 1.03 WHERE `entry` = 8120;
+UPDATE `creature_template` SET `mindmg` = 155, `maxdmg` = 206, `DamageModifier` = 1.03 WHERE `entry` = 7268;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588338165355317100.sql:31]:
```suggestion
UPDATE `creature_template` SET `mindmg` = 176, `maxdmg` = 242, `DamageModifier` = 1.03 WHERE `entry` IN (5649, 8095);
UPDATE `creature_template` SET `mindmg` = 113, `maxdmg` = 170, `DamageModifier` = 1.03 WHERE `entry` = 5648;
UPDATE `creature_template` SET `mindmg` = 161, `maxdmg` = 224, `DamageModifier` = 1.03 WHERE `entry` IN (5650, 7247);
UPDATE `creature_template` SET `mindmg` = 176, `maxdmg` = 234, `DamageModifier` = 1.03 WHERE `entry` = 7269;
UPDATE `creature_template` SET `mindmg` = 149, `maxdmg` = 213, `minrangedmg` = 70, `maxrangedmg` = 112, `DamageModifier` = 1.03 WHERE `entry` = 7246;
UPDATE `creature_template` SET `mindmg` = 112, `maxdmg` = 177, `DamageModifier` = 1.03 WHERE `entry` = 8130;
UPDATE `creature_template` SET `mindmg` = 78, `maxdmg` = 95, `DamageModifier` = 1.03 WHERE `entry` = 7076;
UPDATE `creature_template` SET `mindmg` = 169, `maxdmg` = 204, `DamageModifier` = 1.03 WHERE `entry` = 7286;
UPDATE `creature_template` SET `mindmg` = 186, `maxdmg` = 264, `DamageModifier` = 1.03 WHERE `entry` = 7274;
UPDATE `creature_template` SET `mindmg` = 169, `maxdmg` = 226, `minrangedmg` = 86, `maxrangedmg` = 112, `DamageModifier` = 1.03 WHERE `entry` = 7607;
UPDATE `creature_template` SET `mindmg` = 141, `maxdmg` = 187, `DamageModifier` = 1.03 WHERE `entry` = 7605;
UPDATE `creature_template` SET `mindmg` = 167, `maxdmg` = 213, `DamageModifier` = 1.03 WHERE `entry` = 7608;
UPDATE `creature_template` SET `mindmg` = 122, `maxdmg` = 170, `DamageModifier` = 1.03 WHERE `entry` = 7606;
UPDATE `creature_template` SET `mindmg` = 152, `maxdmg` = 198, `DamageModifier` = 1.03 WHERE `entry` = 7789;
UPDATE `creature_template` SET `mindmg` = 73, `maxdmg` = 96, `DamageModifier` = 1.03 WHERE `entry` = 7788;
UPDATE `creature_template` SET `mindmg` = 88, `maxdmg` = 99, `DamageModifier` = 1.03 WHERE `entry` = 8877;
UPDATE `creature_template` SET `mindmg` = 77, `maxdmg` = 98, `DamageModifier` = 1.03 WHERE `entry` = 7787;
UPDATE `creature_template` SET `mindmg` = 58, `maxdmg` = 87, `DamageModifier` = 1.03 WHERE `entry` = 8876;
UPDATE `creature_template` SET `mindmg` = 182, `maxdmg` = 242, `DamageModifier` = 1.03 WHERE `entry` = 8120;
UPDATE `creature_template` SET `mindmg` = 155, `maxdmg` = 206, `DamageModifier` = 1.03 WHERE `entry` = 7268;
```

**Comment by Geekena** [comment]
all damage and spells are good
i did not found any problem

**Comment by Kitzunu** [comment]
Ty for the PR!

**Pull request merged**